frontend/templates/troubleshooting/index.tsx

```diff
@@ -504,20 +504,26 @@ function NavTabs({
 
    return (
       <Flex {...props} gap={1.5} height="100%">
-         <Link
-            className={devicePartsUrl ? '' : 'isDisabled'}
-            {...notSelectedStyleProps}
-            href={devicePartsUrl}
-         >
-            Parts
-         </Link>
-         <Link
-            className={deviceGuideUrl ? '' : 'isDisabled'}
-            {...notSelectedStyleProps}
-            href={deviceGuideUrl}
-         >
-            Guides
-         </Link>
+         {devicePartsUrl ? (
+            <Link {...notSelectedStyleProps} href={devicePartsUrl}>
+               Parts
+            </Link>
+         ) : (
+            <Box className="isDisabled" {...notSelectedStyleProps}>
+               Parts
+            </Box>
+         )}
+
+         {deviceGuideUrl ? (
+            <Link {...notSelectedStyleProps} href={deviceGuideUrl}>
+               Guides
+            </Link>
+         ) : (
+            <Box className="isDisabled" {...notSelectedStyleProps}>
+               Guides
+            </Box>
+         )}
+
```

Comment:
Completely optional, but we can get rid of the duplicate ternary if we wrap the two links in a react fragment and the two boxes in a react fragment.

Reply:
The conditionals target `devicePartsUrl` and `deviceGuideUrl`, so unfortunately it's not that simple. I was going to get fancy and break this out into a function, but in this instance, I just didn't think it was worth it.